openapi/workflow_execution_service.openapi.yaml

```diff
@@ -820,7 +1291,27 @@ components:
               specification (2.0.0)
         outputs:
           type: object
-          description: The outputs from the workflow run.
+          description: >-
+            The outputs from the workflow run (legacy format). 
+            For outputs that require authentication to access (such as restricted files or directories), 
+            use the 'output_credentials' section of this response to obtain the necessary authentication 
+            credentials. The mapping between specific output files and credentials is implementation-specific 
+            for this legacy format - use 'structured_outputs' for explicit credential references.
+        structured_outputs:
+          $ref: '#/components/schemas/WorkflowOutputs'
+          description: Structured workflow outputs with rich metadata and type safety
+        output_credentials:
```

Comment:
I do not think we should be embedding output credentials in the outputs, ideally we should NEVER have to return credentials but at most use something like a signed URL.

There are a few concerns here
1. Much less secure and exposes access credentials. I think returning DRS ids and then having the caller call each individually is moderately better, but even that pattern is not ideal
2. The engine may not be able to generate output credentials
3. Output credentials can be difficult to produce or lock down scope. IE a passport, bearertoken, api key could allow the user to access much more then JUST the outputs of the current run. A lot of DRS apis returned signed URls but these would be expensive to produce for a large workflow